errors/errors.go

```diff
@@ -80,6 +80,13 @@ var (
 	// or more of the Thema invariants.
 	ErrInvalidLineage = errors.New("invalid lineage")
 
+	// ErrInvalidSchemasOrder indicates that schemas in a lineage are not ordered
+	// by version.
+	ErrInvalidSchemasOrder = errors.New("schemas in lineage are not ordered by version")
+
+	// ErrInvalidLensesOrder indicates that lenses are in the wrong order - they must be sorted by `to`, then `from`.
+	ErrInvalidLensesOrder = errors.New("schemas in lineage are not ordered by version")
```

Comment:
I understand that it should be like this:
```suggestion
	ErrInvalidLensesOrder = errors.New("lenses in schema are not ordered by version")
```

Reply:
Maybe this error could be structured and pass the version as argument.

Reply:
Yes, you are right

Reply:
@spinillos the combination of `from, to` versions for a lens is already passed here https://github.com/grafana/thema/pull/179/files#diff-aade326d3512b5a2ada6faa791ddec468f2a0adedb352339c9e314e74c8949d2R104 - it's done for both current and previous lenses.